### ISSUE TITLE
Not an issue - question on applicability/ future research with music scores / music glyphs

### ISSUE THREAD

**Issue by johndpope** [AIGText/GlyphControl-release#3]
I use an OCR software to recognize music https://www.neuratron.com/photoscore.htm
it fails miserably if the music sheet is a little off due to bad scanning - works perfectly on computer generated pdfs
<img width="1003" alt="Screenshot 2023-06-16 at 5 07 59 am" src="https://github.com/AIGText/GlyphControl-release/assets/289994/0ba31174-6b61-4c30-99f0-65334be75d9e">

I understand that there is music glyphs - https://www.notationcentral.com/product/musglyphs/
so I wonder if this glyph technique / or new research could be done to rebuild scanned pdfs into fresh pdf documents.
in this scenario - text is not so important -  it would be image to image - but I need a base model that is suitably trained on music scores.    Resolutions are a big problem though.

**Comment by PkuRainBow** [comment]
Good point! We would like to see what should we do to help you on generating high-quality music glyphs.

**Comment by johndpope** [comment]
related
https://www.kaggle.com/competitions/denoising-dirty-documents/code
